base-theme/layouts/partials/sentry_capture_message.html

```diff
@@ -1,6 +1,8 @@
 <script type="text/javascript">
   window.addEventListener("load", (event) => {
     console.error("{{ . }}")
-    window.Sentry.captureMessage("{{ . }}")
+    window.Sentry.captureMessage("{{ . }}", {
+      level: Sentry.Severity.Error,
+    })
```

Comment:
The default severity for messages is "Info". Our options are https://develop.sentry.dev/sdk/event-payloads/#optional-attributes _Old versions of sentry had more options, like "critical"._